lnbits/core/views/api.py

```diff
@@ -697,3 +699,34 @@ async def api_auditor(wallet: WalletTypeInfo = Depends(get_key_type)):
         "delta_msats": delta,
         "timestamp": int(time.time()),
     }
+
+
+##################UNIVERSAL WEBSOCKET MANAGER########################
+
+
+@core_app.websocket("/api/v1/ws/{item_id}")
+async def websocket_connect(websocket: WebSocket, item_id: str):
```

Comment:
- nitpick:  `topic_id` instead of `item_id`

Reply:
I think item, but thanks for suggestion!